packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1941,8 +1941,14 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   @protected
   bool hitTestChildren(BoxHitTestResult result, { required Offset position }) {
     final Offset effectivePosition = position - _paintOffset;
-    final InlineSpan? textSpan = _textPainter.text;
-    switch (textSpan?.getSpanForPosition(_textPainter.getPositionForOffset(effectivePosition))) {
+    final GlyphInfo? glyph = _textPainter.getClosestGlyphForOffset(effectivePosition);
+    // This works even with large letter-spacing and text justification, as
+    // graphemeClusterLayoutBounds.width is the x layout advance to the next
```

Comment:
I can guess what x layout advance mean, is it a term that commonly used?

Reply:
In https://freetype.org/freetype2/docs/glyphs/glyphs-3.html it's called advance width. I can change to that if you prefer it over x layout advance?

Reply:
TIL, yes I think using advance width may be more clear